### DISCUSSION TITLE
Why generate "拒绝访问。 (os error 5)“

### DISCUSSION THREAD

**Discussion by kdurant** [BurntSushi/ripgrep#1853] [General] [answered]
```
OS：win10
rg --version
ripgrep 11.0.2 (rev 3de31f7527)
-SIMD -AVX (compiled)
+SIMD +AVX (runtime)
```
![QQ截图20210420134050](https://user-images.githubusercontent.com/2704364/115343173-0c884180-a1de-11eb-94a9-72869fe4ba41.png)

**Comment by BurntSushi** [comment, marked as answer]
I'm sorry, but I don't understand this question. There might be a language barrier here. My suggestion is to do your best to provide a way for others to reproduce your problem.
